src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -107,6 +108,12 @@ private OrbitInfo GetNextPatch()
             return ! GetHasNextPatch() ? null : new OrbitInfo(orbit.nextPatch,Shared);
         }
 
+
+        private double GetNextPatchETA()
+        {
+            return  ! GetHasNextPatch() ? Double.PositiveInfinity : orbit.EndUT - Planetarium.GetUniversalTime();
```

Comment:
On one side I agree with Dunbaratu, you basically want to return something which is outside the "domain" of accepted ETA values. On the other hand, -1 is not a good indication something is wrong straight away. It could screw up calculations further up in the chain and would require a debug session.

So I am not sure of what is best. Is there perhaps a way to throw an exception?